backend/service/k8s/deployment_test.go

```diff
@@ -364,3 +365,569 @@ func TestProtoForDeploymentStatus(t *testing.T) {
 		})
 	}
 }
+
+func TestProtoForDeploymentSpecWithProbesLivenessHTTPGet(t *testing.T) {
+	t.Parallel()
+	var terminationVar int64 = 30
+	var deploymentTestCases = []struct {
+		id         string
+		deployment *appsv1.Deployment
+	}{
```

Comment:
If your going to only have one test case per test func, you can remove all of this extra setup.

Reply:
Remove structure and just keep the deployment